src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    display: inline-block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  .card__info {
+    &-title {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $main-color;
+      margin-bottom: 4px;
+    }
+
+    &-code {
+      font-size: 10px;
+      line-height: 14px;
+      justify-content: flex-start;
+      color: $secondary-color;
+      margin-bottom: 16px;
+    }
+
+    &-rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+
+      &-reviews {
+        font-size: 10px;
+        font-weight: 400;
+        color: $main-color;
+      }
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      font-weight: 700;
+      margin-bottom: 16px;
+
+      &-text {
+        font-size: 12px;
+        color: $secondary-color;
+        font-weight: 400;
+      }
+
+      &-number {
+        font-size: 16px;
+        color: $main-color;
+        line-height: 18px;
+      }
+    }
+  }
+
+  &__button {
+    display: block;
+    height: 40px;
```

Comment:
If you set padding, then in principle paddings are not needed set height here, because the height will be formed from line-height + top/bottom padding